specification/compute/resource-manager/Microsoft.Compute/GalleryRP/stable/2025-03-03/gallery.json

```diff
@@ -2966,6 +3403,244 @@
+        },
+        "supportedOSType": {
+          "type": "string",
+          "description": "This property allows you to specify the supported type of the OS that application is built for. Possible values are: **Windows,** **Linux.**",
+          "enum": [
+            "Windows",
+            "Linux"
+          ],
+          "x-ms-enum": {
+            "name": "OperatingSystemTypes",
+            "modelAsString": false
+          }
+        },
+        "provisioningState": {
+          "readOnly": true,
+          "$ref": "#/definitions/GalleryProvisioningState"
+        }
+      },
+      "required": [
+        "supportedOSType"
+      ],
+      "description": "Describes the properties of a gallery Script Definition."
+    },
+    "GalleryScriptVersion": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/GalleryScriptVersionProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../common-types/v1/common.json#/definitions/Resource"
+        }
+      ],
+      "description": "Specifies information about the gallery Script Version that you want to create or update."
+    },
+    "GalleryScriptVersionUpdate": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/GalleryScriptVersionProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/UpdateResourceDefinition"
+        }
+      ],
+      "description": "Specifies information about the gallery Script Version that you want to update."
+    },
+    "GalleryScriptVersionProperties": {
+      "type": "object",
+      "properties": {
+        "publishingProfile": {
+          "$ref": "#/definitions/GalleryScriptVersionPublishingProfile"
+        },
+        "safetyProfile": {
+          "$ref": "#/definitions/GalleryScriptVersionSafetyProfile"
+        },
+        "provisioningState": {
+          "readOnly": true,
+          "$ref": "#/definitions/GalleryProvisioningState"
+        },
+        "replicationStatus": {
+          "readOnly": true,
+          "$ref": "#/definitions/ReplicationStatus"
+        }
+      },
+      "required": [
+        "publishingProfile"
+      ],
+      "description": "Describes the properties of a gallery script version."
+    },
+    "GalleryScriptVersionPublishingProfile": {
+      "type": "object",
+      "properties": {
+        "source": {
+          "$ref": "#/definitions/ScriptSource"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/GalleryArtifactPublishingProfileBase"
+        }
+      ],
+      "required": [
+        "source"
+      ],
+      "description": "The publishing profile of a gallery image version."
+    },
+    "GalleryScriptVersionSafetyProfile": {
+      "type": "object",
+      "properties": {},
```

Comment:
is this intentional?

Reply:
Yes, it is accurate https://msazure.visualstudio.com/One/_git/Compute-CPlat-PIR?path=/src/CAPS/Service/API/Model/GalleryScriptVersion.cs&version=GBdevelop&line=42&lineEnd=43&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents